pkg/agent/util/net_windows.go

```diff
@@ -63,6 +64,12 @@ func (r Route) String() string {
 		r.LinkIndex, r.DestinationSubnet, r.GatewayAddress, r.RouteMetric)
 }
 
+func (r Route) Equal(x Route) bool {
+	return x.LinkIndex == r.LinkIndex &&
+		x.DestinationSubnet.IP.Equal(r.DestinationSubnet.IP) &&
```

Comment:
Better to check DestinationSubnet is nil or not before use it?

Reply:
Done